openapi/openapi.json

```diff
@@ -25791,6 +25791,20 @@
       "description": "AWSCSIDriverConfigSpec defines properties that can be configured for the AWS CSI driver.",
       "type": "object",
       "properties": {
+        "efsVolumeMetricsFSRateLimit": {
+          "description": "efsVolumeMetricsFSRateLimit defines the rate limit, in goroutines per file system, for processing volume metrics. Requires efsVolumeMetricsOptIn to be enabled. The default rate limit set by the EFS CSI driver is 240 goroutines per file system.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "efsVolumeMetricsOptIn": {
```

Comment:
I am not sure if we need an explicit opt-in boolean, would be `efsVolumeMetricsRefreshPeriod > 0` good enough for an opt-in?

Reply:
Removed this field